package.json

```diff
@@ -7,6 +7,9 @@
 		"type": "git",
 		"url": "https://github.com/DoctorMcKay/node-steam-tradeoffer-manager.git"
 	},
+	"scripts": {
+		"lint": "./node_modules/.bin/eslint ."
```

Comment:
@scholtzm I just add ESlint at the project init and fix the changes as they appear :D
But yeah, thanks for the hint.